README.md

```diff
@@ -1,3 +1,78 @@
-# skywalking-swck
-
 Apache SkyWalking Cloud on Kubernetes
+============
+
+![](https://github.com/apache/skywalking-swck/workflows/Build/badge.svg?branch=master)
+
+<img src="http://skywalking.apache.org/assets/logo.svg" alt="Sky Walking logo" height="90px" align="right" />
+
+A bridge project between [Apache SkyWalking](https://github.com/apache/skywalking) and Kubernetes.
+
+SWCK is a platform for the SkyWalking user, provisions, upgrades, maintains SkyWalking relevant components, and makes them work natively on Kubernetes. 
+
+# Quick Start
+
+ 1. Go to the [download page](https://skywalking.apache.org/downloads/) to download latest release manifest. 
+ 
+ 2. Create the `skywalking-swck-system` namespace:
```

Comment:
`kustomize build config/default > ${bindir}/config/deploy.yaml` contains create`namespace`, I think we can skip this step :)

Reply:
done